### ISSUE TITLE
Fragment shader seems not executed

### ISSUE THREAD

**Issue by manhnt9** [GPUOpen-LibrariesAndSDKs/V-EZ#52]
I'm trying to render a colored triangle.
Here is my shaders:
```cpp
  shaderInfo.VSSource =
    "#version 450\n"
    "#extension GL_ARB_separate_shader_objects : enable\n"
    "layout (location = 0) in vec4 position;\n"
    "void main() {\n"
    "  gl_Position = position;\n"
    "}\n";

  shaderInfo.PSSource =
    "#version 450\n"
    "#extension GL_ARB_separate_shader_objects : enable\n"
    "layout (location = 0) out vec4 frag_color;\n"
    "void main() {\n"
    "  frag_color = vec4(0.0, 1.0, 0.0, 1.0);\n"
    "}\n";
```
Both shaders are created and included in pipeline creation.
Screen output is only a solid color (clear color).
Using RenderDoc, I found out that the vertices are rendered correctly.

![screenshot_2018-10-05_09-45-19](https://user-images.githubusercontent.com/11463928/46513563-65439900-c883-11e8-9484-d99d4e079405.png)

![screenshot_2018-10-05_09-40-59](https://user-images.githubusercontent.com/11463928/46513465-cb7bec00-c882-11e8-8baf-21fbb5e4323e.png)

And there's no Fragment Shader Output tab in RenderDoc for me to see.
So I think it's not executed.
Is there some step that I am missing?
My capture file: https://drive.google.com/open?id=1jh1YtqKVdm8P3NEYryRPaQqksky0Vyrq

**Comment by soconne** [comment]
Can you send me a zip file with your full source code that I can compile and run on my end?

**Comment by manhnt9** [comment]
I've sent a version to your mail showed on your github profile.

**Comment by manhnt9** [comment]
Solved by changing the front face mode to
```cpp
rasterizationState.frontFace = VK_FRONT_FACE_COUNTER_CLOCKWISE;
```
Suggest by one friend of mine.
Vulkan is using right handed coordinate system.
Another example that I tried was loading mesh, I had to negate y components of position and normals too.

Triangle demo:
![screenshot_2018-10-06_23-39-04](https://user-images.githubusercontent.com/11463928/46573539-04ba7600-c9c1-11e8-9ee5-43f9f7df3bf3.png)

Teapot model demo:
![screenshot_2018-10-06_23-39-28](https://user-images.githubusercontent.com/11463928/46573545-13089200-c9c1-11e8-86a1-f3950b7276e7.png)
